lib/common/src/Amp.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+namespace Amp;
+
+use DOMElement;
+use DOMNode;
+
+/**
+ * Central helper functionality for all Amp-related PHP code.
+ *
+ * @package amp/common
+ */
+final class Amp
+{
+
+    /**
+     * List of Amp attribute tags that can be appended to the <html> element.
+     *
+     * @var string[]
+     */
+    const TAGS = ['amp', '⚡', '⚡4ads', 'amp4ads', '⚡4email', 'amp4email'];
```

Comment:
One more is needed here. See https://github.com/ampproject/amphtml/issues/25990

Reply:
I haven't yet looked into this, because there was discussion about supporting all 16 variations instead of only 2. But it looks like only 2 are supported by the validator for now, so I can adapt to this now.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/4019/commits/6af5a580200b67b1a581502813698ffee1a9feda